uber/site_sections/preregistration.py

```diff
@@ -508,7 +508,7 @@ def pay_for_extra_codes(self, session, payment_id, stripeToken):
     def group_members(self, session, id, message='', **params):
         group = session.group(id)
         charge = Charge(group)
-        if group.status != c.APPROVED and 'name' in params:
+        if cherrypy.request.method == 'POST':
```

Comment:
I think I may have merged this wrong. It should probably still check the approval status... I'll have to test it.

Reply:
Turns out this was caught by a model check, which I'll improve in the next commit.